--- Chromium-0.9/src/define.h.config	Sat Feb 10 03:33:24 2001
+++ Chromium-0.9/src/define.h	Thu May 24 19:50:40 2001
@@ -42,12 +42,14 @@
 
 #define MAX_BUFF 256
 
+#define PKGBINDIR "/usr/bin"
 #ifdef PKGBINDIR
 #define CHROMIUM_EXEC PKGBINDIR"/chromium"
 #else 
 #define CHROMIUM_EXEC "./chromium"
 #endif //PKGBINDIR
 
+#define PKGDATADIR "/usr/games/chromium"
 #ifdef PKGDATADIR
 #define CHROMIUM_DATA_DIR PKGDATADIR"/data"
 #else
--- Chromium-0.9/scripts/config.std.config	Thu Feb  8 06:18:26 2001
+++ Chromium-0.9/scripts/config.std	Thu May 24 19:48:17 2001
@@ -5,7 +5,7 @@
 
 SDL_CFLAGS		= `$(SDL_CONFIG) --cflags`
 SMPEG_CFLAGS	= 
-PKG_FLAGS		= 
+PKG_FLAGS		= $(RPM_OPT_FLAGS)
 
 GL_LIBS		= -lglpng -lGL -lGLU -lm
 AL_LIBS		= -lopenal 
--- Chromium-0.9/scripts/Makefile.all.config	Sat Feb 10 03:32:31 2001
+++ Chromium-0.9/scripts/Makefile.all	Thu May 24 19:48:17 2001
@@ -35,6 +35,13 @@
 	-rm -f *~ core
 	-rm -f *.bck
 
+install:
+	mkdir -p $(RPM_BUILD_ROOT)$(bindir)
+	mkdir -p $(RPM_BUILD_ROOT)/$(prefix)/games/chromium/data
+	install -c -s ./bin/* $(RPM_BUILD_ROOT)$(bindir)/
+	(cd Chromium-0.9; find -name CVS | xargs rm -rf)
+	cp -rf Chromium-0.9/data $(RPM_BUILD_ROOT)$(prefix)/games/chromium/
+	
 distclean: clean
 	@echo ""
 	-rm -f scripts/*.bck
--- Chromium-0.9/configure.config	Thu May 24 19:54:03 2001
+++ Chromium-0.9/configure	Thu May 24 19:53:36 2001
@@ -138,7 +138,7 @@
 	
 ## OpenAL configure options
 ######################################################################
-OPENAL_CONFIG_OPTS="./configure"
+OPENAL_CONFIG_OPTS="CFLAGS=\"$RPM_OPT_FLAGS\" ./configure"
 
 # are we using SDL?
 if [ $ENABLE_SDL = "yes" ]; then
