--- chromium-bsu-0.9.14/configure.ac~	2009-09-22 09:42:08.173550200 +0200
+++ chromium-bsu-0.9.14/configure.ac	2009-09-22 09:44:35.873535456 +0200
@@ -143,7 +143,7 @@
 fi
 
 if test "x$TEXT_TYPE" = xGLC -a "x$FONT_NAME" != x ; then
-	TEXT_CFLAGS+=" -DFONT_NAME=\"\\\"$FONT_NAME\\\"\""
+	TEXT_CFLAGS="$TEXT_CFLAGS  -DFONT_NAME=\"\\\"$FONT_NAME\\\"\""
 	AC_SUBST(TEXT_CFLAGS)
 	FONT="$FONT_NAME"
 elif test "x$TEXT_TYPE" = xGLC ; then
@@ -151,18 +151,18 @@
 fi
 
 if test "x$TEXT_TYPE" = xFTGL -a "x$FONT_PATH" != x ; then
-	TEXT_CFLAGS+=" -DFONT_PATH=\"\\\"$FONT_PATH\\\"\""
+	TEXT_CFLAGS="$TEXT_CFLAGS  -DFONT_PATH=\"\\\"$FONT_PATH\\\"\""
 	AC_SUBST(TEXT_CFLAGS)
 	FONT="$FONT_PATH "
 fi
 
 if test "x$TEXT_TYPE" = xFTGL -a "x$HAVE_FONTCONFIG" = xyes ; then
 	AC_DEFINE(HAVE_FONTCONFIG, 1, [define to use fontconfig])
-	TEXT_CFLAGS+=" $FONTCONFIG_CFLAGS"
+	TEXT_CFLAGS="$TEXT_CFLAGS  $FONTCONFIG_CFLAGS"
 	AC_SUBST(TEXT_CFLAGS)
-	TEXT_LIBS+=" $FONTCONFIG_LIBS"
+	TEXT_LIBS="$TEXT_LIBS  $FONTCONFIG_LIBS"
 	AC_SUBST(TEXT_LIBS)
-	FONT+="fontconfig (bold Gothic Uralic) "
+	FONT="$FONT fontconfig (bold Gothic Uralic) "
 fi
 
 if  test "x$TEXT_TYPE" = xFTGL -a "x$HAVE_FONTCONFIG" != xyes -a "x$FONT_PATH" = x ; then
@@ -189,7 +189,7 @@
 
 if test "x$HAVE_SDLIMAGE" = xyes -a "x$WINDOW_TYPE" = xSDL -a "x$enable_sdlimage" != xno; then
 	AC_DEFINE(ENABLE_WINDOW_ICON, 1, [define to enable the window icon (needs SDL_image)])
-	WINDOW_LIBS+=-lSDL_image
+	WINDOW_LIBS="$WINDOW_LIBS -lSDL_image"
 	AC_SUBST(WINDOW_LIBS)
 	WINDOW_ICON=yes
 else
