diff -Nru Chromium-0.9/src-setup/Makefile Chromium-0.9-new/src-setup/Makefile
--- Chromium-0.9/src-setup/Makefile	Sat Dec 22 03:03:31 2001
+++ Chromium-0.9-new/src-setup/Makefile	Sat Dec 22 03:04:18 2001
@@ -12,7 +12,7 @@
 CXX	=	g++
 #CFLAGS	=	-pipe $(PKG_CFLAGS) $(AL_CFLAGS) $(SDL_CFLAGS) $(SMPEG_CFLAGS) -Wall -W -g -D_REENTRANT
 #CXXFLAGS=	-pipe $(PKG_CFLAGS) $(AL_CFLAGS) $(SDL_CFLAGS) $(SMPEG_CFLAGS) -Wall -W -g -D_REENTRANT
-INCPATH	=	-I$(QTDIR)/include
+INCPATH	=	-I$(QTDIR)/include/qt
 LINK	=	g++
 LFLAGS	=	
 LIBS	=	$(SUBLIBS) -L$(QTDIR)/lib -L/usr/X11R6/lib  $(AL_LIBS) $(SDL_LIBS) $(SMPEG_LIBS) $(VORBIS_LIBS) -lm -pthread -lqt -lXext -lX11 -lm
