--- ./support/openal/linux/configure.in.org	Fri Sep  6 22:28:16 2002
+++ ./support/openal/linux/configure.in	Fri Sep  6 22:28:26 2002
@@ -133,10 +133,15 @@
 AC_C_BIGENDIAN
 
 dnl I'm afraid we need libm
-AC_CHECK_LIB(m, sin, LIBS="$LIBS -lm", echo "I'm afraid you'll need libm."; exit 1)
+AC_CHECK_LIB(m, sin, 
+	     [LIBS="$LIBS -lm"], 
+	     [AC_MSG_ERROR("I'm afraid you'll need libm.") ])
 
 dnl Should be allow dlopen of extensions?
-AC_CHECK_LIB(dl, dlopen, LIBS="$LIBS -ldl", AC_DEFINE(NODLOPEN) echo "Can't dlopen extensions." )
+AC_CHECK_LIB(dl, dlopen, 
+	     [LIBS="$LIBS -ldl"], 
+	     [AC_DEFINE(NODLOPEN) 
+	      AC_MSG_WARN("Can't dlopen extensions.") ])
 
 dnl Special OS objs for target systems
 case "$target" in
@@ -215,28 +220,27 @@
 fi
 
 dnl dmalloc stuff
-AC_ARG_ENABLE(dmalloc,
-[  --enable-dmalloc          enable DMALLOC support       [default=no]],
+AC_ARG_ENABLE(dmalloc,AC_HELP_STRING([--enable-dmalloc],[enable DMALLOC support ]),
               , enable_dmalloc=no)
 if test x$enable_dmalloc = xyes; then
     AC_CHECK_HEADER(dmalloc.h, 
-    	dnl header found
+    	[dnl header found
 	AC_DEFINE(DMALLOC)
 	AC_DEFINE(DMALLOC_FUNC_CHECK)
     	INCLUDES="$INCLUDES -include \"/usr/include/dmalloc.h\""
     	LIBS="$LIBS -ldmallocth"
 	SHOULD_USE_WERROR=no
-    	REDUNDANT_DECLS_WARN=no,
-	dnl header not found, check local/include
+    	REDUNDANT_DECLS_WARN=no],
+	[dnl header not found, check local/include
 	dnl invalid cache
 	AC_CHECK_HEADER(/usr/local/include/dmalloc.h,
-		dnl header found in local/include
+		[dnl header found in local/include
 		AC_DEFINE(DMALLOC)
 		AC_DEFINE(DMALLOC_FUNC_CHECK)
 		INCLUDES="$CFLAGS -include \"/usr/local/include/dmalloc.h\""
 		LIBS="$LIBS -ldmallocth"
 		SHOULD_USE_WERROR=no
-		REDUNDANT_DECLS_WARN=no))
+		REDUNDANT_DECLS_WARN=no])])
 fi
 
 dnl efence stuff
