--- scripts/Makefile.all.orig	2003-11-19 19:41:22.502256736 -0500
+++ scripts/Makefile.all	2003-11-19 19:41:25.414813960 -0500
@@ -1,5 +1,4 @@
 SUPPORT_DIRS	=	 \
-			./support/openal/ \
 			./support/glpng/src
 
 CHROMIUM_DIRS	=	\
--- scripts/Makefile.no-setup.orig	2003-11-19 19:41:22.502256736 -0500
+++ scripts/Makefile.no-setup	2003-11-19 19:41:25.414813960 -0500
@@ -1,5 +1,4 @@
 SUPPORT_DIRS	=	 \
-			./support/openal/ \
 			./support/glpng/src
 
 CHROMIUM_DIRS	=	\
--- src-setup/Audio.cpp.orig	2003-11-19 20:03:23.237474464 -0500
+++ src-setup/Audio.cpp	2003-11-19 20:03:49.838430504 -0500
@@ -71,7 +71,14 @@
 	if(!initialized)
 	{
 		fprintf(stderr, "starting OpenAL...");
+#ifdef OLD_OPENAL
 		context_id = alcCreateContext(NULL);
+#else
+		ALCdevice *dev;
+		dev = alcOpenDevice( NULL );
+		context_id = alcCreateContext(dev, NULL);
+		alcMakeContextCurrent(context_id);
+#endif
 		initialized = (bool)context_id;
 		fprintf(stderr, "done.\n");
 
